placeSpecificContent/hq/courtyard/entityResources/podium/courtyardPodium_server.js

```diff
@@ -0,0 +1,173 @@
+//
+//  courtyardPodium_server.js
+//
+//  Created by Rebecca Stankus on 07/16/19
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+(function() { 
+    var _this;
+
+    var MOVEMENT_VELOCITY_M_PER_SEC = 1.4;
+    var POSITION_CHECK_INTERVAL_MS = 25;
+    var OPEN_SOUND = SoundCache.getSound(Script.resolvePath("sounds/openGate.wav"));
+    var EPSILON_M = 0.01;
+
+    var gatePositionCheckInterval;
+    var openedLocalPosition;
+    var closedLocalPosition;
+
+    // *************************************
+    // START UTILITY FUNCTIONS
+    // *************************************
+
+    /* PLAY A SOUND: Plays the specified sound at specified position and volume */
+    var injector;
+    function playSound(sound, position, volume) {
+        if (sound.downloaded) {
+            if (injector) {
+                injector.stop();
+            }
+            
+            var gameAudioPosition =
+                Script.require(Script.resolvePath('../../config/config.json?' + Date.now())).gameAudioPosition;
+            injector = Audio.playSound(sound, {
+                position: gameAudioPosition,
+                volume: volume
+            });
+        }
+    }
+
+    // *************************************
+    // END UTILITY FUNCTIONS
+    // *************************************
+
+    var Gate = function() {
+        _this = this;
+    };
+
+    Gate.prototype = {
+        remotelyCallable: ['openGate', 'closeGate'],
+
+        // Sets up this entity's closed and opened positions
+        preload: function(entityID){
+            _this.entityID = entityID;
+            var name = Entities.getEntityProperties(_this.entityID, 'name').name;
+            if (name === "Bingo Scanner Entry Gate") {
```

Comment:
I think these Bingo things are left over?

Reply:
Sorry, this was not supposed to be included. I ended up not having a server file on the podium.